### ISSUE TITLE
iOS App crashes on launch without any error

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#12199]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1150610/ios-app-crashes-on-lunch-without-any-error.html)._

---
<p>This started suddenly on last version of VS for Mac 8.7 (build 2037)  and it keeps in current version 8.7.2 (build 4), affecting iOS 11.3 and newer. </p><p>There is no any log giving any reference about what is happen. I'm attaching a video trying to demonstrate.</p><p></p><p><a href="https://developercommunity.visualstudio.com/storage/attachments/151618-bug-iosmov.zip">bug-iosmov.zip</a></p>

---
#### Wallace Cezar Sales dos Santos on 8/14/2020, 07:37 AM: 

(private comment, text removed)
#### Feedback Bot on 8/17/2020, 01:32 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

#### Wallace Santos on 8/28/2020, 05:26 AM: 

(private comment, text removed)
#### Whitney Schmidt [MSFT] on 8/28/2020, 10:40 AM: 

<p>@<a target='_blank' href="https://developercommunity.visualstudio.com/users/80039/WallaceCezarSalesdosSantos.html">Wallace Cezar Sales dos Santos</a> Thank you for your feedback and the descriptive video!</p>
<p>It’s hard to diagnose this issue without more information. A couple of things to try:</p>
<ol>
<li>
<p>Could you try creating a default iOS application and building/running it? If it doesn’t crash, that would point towards something specific inside of your project. If it does, then this might be related to the IDE.</p>
</li>
<li>
<p>If it’s possible, could you create a small sample project and upload it here? You can upload files and add comments that are private (not visible to the dev community, but visible to internal folks) if needed. The best way for us to investigate is to have a solution that allows us to reproduce the issue locally.</p>
</li>
<li>
<p>You shared some of your version information, could you share all of it with the following steps?</p>
</li>
</ol>
<p>The easiest way to get exact version information:</p>
<ul>
<li>On Visual Studio for Mac: “Visual Studio” menu, “About Visual Studio” item, “Show Details” button.</li>
<li>On Visual Studio for Windows: “Help menu”, “About Microsoft Visual Studio” item.</li>
</ul>
<p>Then copy/paste the version information (you can use the “Copy Information” button).</p>
<p>We look forward to hearing from you!</p>

#### Wallace Santos on 8/28/2020, 02:11 PM: 

(private comment, text removed)
#### Whitney Schmidt [MSFT] on 9/3/2020, 08:28 AM: 

<p>@<a target='_blank' href="https://developercommunity.visualstudio.com/users/80039/WallaceCezarSalesdosSantos.html">Wallace Cezar Sales dos Santos</a> Thanks for the update - any news on your investigation?</p>

#### Wallace Santos on 9/3/2020, 01:00 PM: 

<p><a rel="user" href="https://developercommunity.visualstudio.com/users/187437/372a51b5-eb01-49bd-831e-d2d249c78ed6.html" nodeid="187437">@Whitney Schmidt [MSFT]</a> unfortunately, not yet. I want to try some else before to give up. I'll keep you posted.</p>

#### Rolf Kvinge [MSFT] on 9/9/2020, 07:26 AM: 

<p>@<a target='_blank' href="https://developercommunity.visualstudio.com/users/80039/WallaceCezarSalesdosSantos.html">Wallace Cezar Sales dos Santos</a>: Were you able to figure anything out? If not, could you check if there any new files in ~/Library/Logs/DiagnosticReports when the app disappears? Usually this is where crash reports end up for apps that crash in the simulator.</p>

#### Wallace Santos on 9/9/2020, 07:46 AM: 

(private comment, text removed)
#### Wallace Santos on 9/18/2020, 06:13 AM: 

(private comment, text removed)
#### Rolf Kvinge [MSFT] on 9/18/2020, 06:19 AM: 

<p>OK, thanks for the investigation! I’ll reassign this to the Xamarin.Forms team, and they will get back to you if either more information is needed, a workaround is available, or the issue is resolved.</p>

#### Wallace Santos on 9/21/2020, 03:09 AM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by themronion** [comment]
What about breakfast? Does it crash only on lunch or maybe on dinner too?

**Comment by rmarinho** [comment]
Hi , can you please upload a small reproduction of the issue so we can try to understand what's happening?!

Thanks

**Comment by RGBPlaza** [comment]
My app is crashing not only for breakfast, lunch and dinner, but also elevenses!!! I'm meant to be rolling out in two days!!!

**Comment by samhouts** [comment]
Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by RGBPlaza** [comment]
Well for me it turns out it was a problem with the shared transitions library which I believe has now been fixed...

**Comment by themronion** [comment]
@RGBPlaza close the issue then)

**Comment by RGBPlaza** [comment]
I didn't create the issue - I don't think I can do anything about it...

**Comment by samhouts** [comment]
Okay, thanks @RGBPlaza ! We'll wait for the original poster for a bit longer.

**Comment by Wallysantos** [comment]
@samhouts and @rmarinho the specific crash showed in the video, as @RGBPlaza commented, was fixed, but if the end of the original thread, I pointed another situation while using an Entry is property IsPassword = true in iOS. I don't believe that the problem is with Xamarin Forms, but maybe when Materials are applied, something happen. I already created a workaround in my project to solve this situation.
